docs/rules/sort-comp.md

```diff
@@ -171,6 +172,48 @@ var Hello = React.createClass({
 });
 ```
 
+If you want to flow annotations to be at the top:
+
+```js
+"react/sort-comp": [1, {
+  order: [
+    'type-annotations',
+    'static-methods',
+    'lifecycle',
+    'everything-else',
+    'render',
+  ],
+}]
+```
+
+With the above configuration, the following patterns are considered warnings:
+
+```js
+class Hello extends React.Component<any, Props, void> {
+  onClick() { this._someElem = true; }
+  props: Props;
```

Comment:
what does `props: Props` desugar to in Flow? Does this whole line just get removed, because it's just a type declaration, or does it have another effect?